src/administration/integrations/gluescript.md

```diff
@@ -0,0 +1,251 @@
+>
+> GlueScript scripts currently do not "live in Git". They are project wide, and don't belong to a specific branch. But it is a good practice to consider that whatever you put there in your `master` branch will be what will have been posted to the API. Later we will introduce a more robust management mechanism.
+
+#### Example: Custom Slack integration
+
+For example, a GlueScript can perform a custom Slack integration as follows:
+
+```javascript
+var message = activity.text;
+var color;
+
+if (activity.state == 'complete') {
+  if (activity.result == 'success') {
+    color = '#66c000';
+  } else {
+    color = '#ff0000';
+  }
+}
+
+var resp = fetch(
+  '<SLACK_PAYLOAD_URL>',
+  {
+    method: 'POST',
+    headers: {
+      'Content-Type': 'application/json',
+    },
+    body: JSON.stringify({
+      'attachments': [{
+        'fallback': message,
+        'color': color,
+        'fields': [{
+          'value': message,
+        }],
+      }],
+    }),
+  }
+);
+
+if (!resp.ok) { throw 'Failed'; }
+```
+
+where:
+* `SLACK_PAYLOAD_URL` is the URL provided by Slack for your custom webhook
+
+You can save the file to `.platform/scripts/my_glue_script.js` for example.
+
+### 2. Commit the script
+
+Commit the script to Git:
+
+```bash
+git add .platform/scripts/my_glue_script.js
+git commit -m "Committing the GlueScript file."
+git push platform master
+```
+
+where:
+* `my_glue_script.js` is the name of your Javascript file
+
+> Pushing the script to master won't do anything. Yet. Just a handy place to keep your GlueScript file.
+
+### 3. Enable the integration
+
+Enable the GlueScript integration as follows (you need to have the Platform.sh CLI installed):
+
+```bash
+platform integration:add --type script --file .platform/scripts/my_glue_script.js
+```
+
+where:
+* `my_glue_script.js` is the name of your Javascript file
+
+> **note**
+> This is the actual activation of the integration, the script is posted to the API through the CLI. Note that changing the file in Git will not change the integration. To update the integration you will need to use the CLI again.
+>
+> The GlueScript integration can only be managed by the project admin.
+
+# GlueScript APIs
+
+These are the internal Platform.sh APIs that are accessible through GlueScript, API calls are executed with the permissions of the user that created the integration.
+
+## Project API
+
+The project API gives you everything you need to know about the project in the context of which the GlueScript is running. 
+
+For example: `project.environments.master` will give you access to the metadata of the master environment of the current project.
+
+The API starts at `api.project` and is based on a one resource == one object model.
+
+### Properties
+
+The properties of the resource can be access directly on the object, like:
+
+```javascript
+project.owner
+```
+
+### Sub-objects
```

Comment:
I feel that the sub-object section is redundant because we're drawing an arbitrary distinction between properties that are primitive data types vs. properties that are objects. Sure, `project.owner` is a string property, but `project.settings` is also a property that happens to have its own properties.